### PR TITLE
Release 11.0.0 branch

### PR DESCRIPTION
**Pull request by caspahouzer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,200 @@
...
+  * problems with "userinterfacestyle" event ([bdfd640](https://github.com/tidev/titanium_mobile/commit/bdfd64029484abd2f3b61b1239c6cd3835e82722))
+  * [TIMOB-28563](https://jira-archive.titaniumsdk.com/TIMOB-28563) - use Activity for colors resolution ([e290733](https://github.com/tidev/titanium_mobile/commit/e2907331ef681ca2aaf6d96d1e1718dbeb427acc))
+  * reuse address for debugger ([c404163](https://github.com/tidev/titanium_mobile/commit/c4041636297034167bc581474c7057acf5ebae7f))
+  * [TIMOB-28583](https://jira-archive.titaniumsdk.com/TIMOB-28583) - commit fragments transaction only once after wrapping view being attached to a window ([0838db2](https://github.com/tidev/titanium_mobile/commit/0838db24a64d0eec534d02b9b43efea8d98d7d05))
+  * crash in ListView on API 21 ([b30de10](https://github.com/tidev/titanium_mobile/commit/b30de101032fc668bcf8284559aec3f0a4eacb52))
+  * Titanium.UI.iOS.Toolbar was removed in 10.0.0 ([23986a3](https://github.com/tidev/titanium_mobile/commit/23986a36be11b52a8485ce8971e6e3869c7bc738))
+
+* m1ga
+  * update documentation links in readme ([79a94f7](https://github.com/tidev/titanium_mobile/commit/79a94f746b4971af4b10416761b779c8a431ed04))
```
Comment (m1ga) [inline CHANGELOG.md:86]: can you move that into my other section above? Not sure why this is separate.

Comment (caspahouzer) [reply CHANGELOG.md:86]: I already did in my next PR for the official release note in the docs, but I can also do it here

Comment (m1ga) [reply CHANGELOG.md:86]: ah no problem!
just wanted to add that you'll need to look at https://github.com/tidev/titanium-docs/tree/main/docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_10.x too :smile: 
And keep in mind: there will be a RC before the GA so it won't be the official release

**Review by cb1kenobi — APPROVED** [review]
LGTM **APPROVED**

**Pull request merged**